packages/server/src/automations/triggerInfo/index.js

```diff
@@ -0,0 +1,15 @@
+const app = require("./app")
+const cron = require("./cron")
+const rowDeleted = require("./rowDeleted")
+const rowSaved = require("./rowSaved")
+const rowUpdated = require("./rowUpdated")
+const webhook = require("./webhook")
+
+exports.definitions = {
+  ROW_SAVED: rowSaved.definition,
+  ROW_UPDATED: rowUpdated.definition,
+  ROW_DELETED: rowDeleted.definition,
+  WEBHOOK: webhook.definition,
+  APP: app.definition,
+  CRON: cron.defintion,
```

Comment:
actually it will still work because it's been imported correctly 😂 but the same typo here